dashboard/modules/snapshot/snapshot_head.py

```diff
@@ -98,7 +98,9 @@ async def get_actor_info(self):
         actors = {}
         for actor_table_entry in reply.actor_table_data:
             actor_id = actor_table_entry.actor_id.hex()
-            runtime_env = json.loads(actor_table_entry.serialized_runtime_env)
+            task_spec = actor_table_entry.task_spec
```

Comment:
Wait I don't think this actually works for dead actors since we clear the task spec (see #11149)?

Reply:
I see, very strange...

Reply:
reverting change